web/src/features/panels/zone/ZoneHeaderTitle.tsx

```diff
@@ -76,9 +78,11 @@ export default function ZoneHeaderTitle({ zoneId }: ZoneHeaderTitleProps) {
         </div>
         <TimeDisplay className="whitespace-nowrap text-sm" />
       </div>
-      <div className="ml-auto self-center px-3">
-        <ShareButton />
-      </div>
+      {isShareButtonEnabled && (
+        <div className="ml-auto self-center px-3">
+          <ShareButton />
+        </div>
```

Comment:
Is this div really needed?

I'd think you can add the classes directly to the button?
Or even extract the alignment to the outer flexbox.

Reply:
That's a great point, but let's take it over to the base branch: https://github.com/electricitymaps/electricitymaps-contrib/pull/7240